src/lib/utils.js

```diff
@@ -267,26 +267,26 @@ export function getSharedLayersFromURL(mapId) {
  * @param {*} groupName Name of the group which we want to target
  * @param {*} children Children of the group which we want to target
  */
-export function getMenuGroupMapLayers(groupName, children) {
-  const subGroups = children.filter(child => !child.id);
-
-  if (subGroups.length) {
-    let visibleLayerIds = [];
-
-    subGroups.forEach(sg => {
-      Object.keys(sg).forEach(key => {
-        const subGroupVisibleLayerIds = getMenuGroupMapLayers(groupName, sg[key].layers);
-        visibleLayerIds = [...visibleLayerIds, ...subGroupVisibleLayerIds];
-      });
-    });
-
-    // Return all layer Ids of the nested subgroups
-    return visibleLayerIds;
-  } else {
-    // Return the Ids of layers for the group
-    return children.map(child => child.id);
-  }
-}
+// export function getMenuGroupMapLayers(groupName, children) {
```

Comment:
Why was this commented? I can see the functionality has been instead copied into the function `getCategoryForLayers`